Compilation error when TLS_DEBUGGING is set

When -DTLS_DEBUGGING is set, the following compilation error occurs

```
 tls.c:636:13: error: use of undeclared identifier 'fd'
  636 |             fd, conn->c.state, mask, conn->c.read_handler != NULL, conn->c.write_handler != NULL,
      |             ^
```

--- a/src/tls.c
+++ b/src/tls.c
@@ -633,7 +633,7 @@ static void tlsHandleEvent(tls_connection *conn, int mask) {
     int ret, conn_error;
 
     TLSCONN_DEBUG("tlsEventHandler(): fd=%d, state=%d, mask=%d, r=%d, w=%d, flags=%d",
-            fd, conn->c.state, mask, conn->c.read_handler != NULL, conn->c.write_handler != NULL,
+            conn->c.fd, conn->c.state, mask, conn->c.read_handler != NULL, conn->c.write_handler != NULL,
             conn->flags);
 
     ERR_clear_error();

